packages/postcss-discard-unused/src/index.js

```diff
@@ -8,6 +8,11 @@ const atrule = 'atrule';
 const decl = 'decl';
 const rule = 'rule';
 
+function normalizeFontName (value) {
+    // Ignore casing and wrapping quotes when checking for font use.
+    return value.toLowerCase().replace(/^\s*(['"])(.*?)(\1)\s*$/, '$2');
+}
```

Comment:
Bad idea, it is wrong, you can find how normalize font-name in other our plugin

Reply:
What plugin is that? There are quite a few in this repo.

Reply:
Here right solution https://github.com/cssnano/cssnano/blob/a520e6906e7fa17951a64769f030ed7b6f44c38a/packages/postcss-minify-font-values/src/lib/minify-family.js, it is really not easy parse font-family, maybe we should move this utils to separate package

Reply:
In what scenario do you see this failing? The only scenario I can think of would be if there were a font-face with a name that matched a non font-family value in a font property, something like `bold` or `10px`. That would be a very unusual edge case, and likely an invalid font name. Even then, the failure would be that the poorly named font-face would not be discarded.

Reply:
It it nor poor, some font can have very weird characters in name, so we should parse this as w3 say

Reply:
According to [the w3's font shorthand documentation](https://www.w3.org/TR/CSS2/fonts.html#font-shorthand), the font shorthand must always include a font-size and font-family/line-height as the last two properties. Knowing this, it is actually pretty simple to extract the font-family value from a font shorthand. All we have to do is identify the last matching font-size/line-height property and strip it and everything before it out of the font value. I added a method `getFontFamilyFromShorthand` which does just that. I thin the current iteration of this change is pretty solid. Let me know what you think.

Reply:
@GreenGremlin we should not use `regexp` for normalize, problem not in shorthard, problem what font name can contains unicode/"/' character and you regexp is broken in this case, we should use parser for this as i describe above

Reply:
Maybe "normalize" isn't the best name for what this fix does. All this fix is trying to do is strip out wrapping quotes and escape characters from font names so that they can be compared in a base format. We really shouldn't need to fully "normalize" the fonts, like is done in the postcss-minify-font-values. I did rework my changes using postcss-value-parser, which makes the parsing a bit more clean. I also broke the font name parsing logic out to its own module, with its own tests. If you still think there are some unicode characters that will break this, please give me an example and I'll add a test for it.